sycl/doc/extensions/experimental/sycl_ext_oneapi_properties.asciidoc

```diff
@@ -80,9 +80,61 @@ properties P1{bar};
 properties P2{bar, foo{argv[0]}};
 properties P3{bar, baz};
 properties P4{baz, bar};
+
 static_assert(!std::is_same_v<decltype(P1), decltype(P2)>); // P1 and P2 contain different properties
 static_assert(!std::is_same_v<decltype(P1), decltype(P3)>); // P1 and P3 contain different properties
 static_assert(std::is_same_v<decltype(P3), decltype(P4)>); // The order properties are specified doesn't change the type
+
+```
+
+The alias `sycl::ext::oneapi::experimental::properties_t` is defined for
+convenience and it is defined as follows:
+
+```c++
+namespace sycl::ext::oneapi::experimental {
+
+template <auto... Props> using properties_t = decltype(properties{Props...});
+
+} // namespace experimental::oneapi::ext::sycl
```

Comment:
These additions are to the "Examples" section, which is not a formal part of this specification.  The formal definition of `properties_t` should be added someplace in the "Specification" section, probably under "Type of properties".

I also think you  have added too much information about `properties_t` to this introductory example.  This section is the general motivation for the entire compile-time properties extension, and half of the example is now devoted to `properties_t`.  That seems disproportionate.

Do we need anything in this introductory section on `properties_t`?  It seems like a secondary feature to me, which can be described in the main part of the specification.  For instance, the example could be moved below to the "Type of properties" section.

Reply:
I created a separate section right after "Type of properties" section and moved everything there.